### ISSUE TITLE
Preview of some letter images broken

### ISSUE THREAD

**Issue by MonikaBarget** [Maynooth-Center-for-Digital-Humanities/omeka-deploy#3]
Some letter images in OMEKA have no preview at all: cf. letter #4722. Wrong upload format?

**Comment by fabianoP** [comment]
Cannot find letter #4722 was it deleted?

**Comment by MonikaBarget** [comment]
Example:

![image](https://user-images.githubusercontent.com/38257338/39179756-fafc83b0-47ac-11e8-8c8a-8c54cf085a1c.png)

**Comment by MonikaBarget** [comment]
Yes, the uploader seems to have deleted image 4722, but many other images have the same issue.

**Comment by MonikaBarget** [comment]
http://xl1916apps.mucampus.ie/diyhistory/admin/items/show/4717

**Comment by MonikaBarget** [comment]
This is the file information of one of the images which isn't displayed correctly: http://xl1916apps.mucampus.ie/diyhistory/admin/files/show/11396

![image](https://user-images.githubusercontent.com/38257338/39182806-f72f8904-47b5-11e8-8934-51be52266cfa.png)


It seem that the contributor uploaded a JPEG file, and the size shows that the file was not "empty". If the file is too large for upload or if some conversion error occurred, we should maybe implement a feature in the upload form that warns contributors before they press the "submit" button. In cases like these, we have no possibility at all to retrieve the images ourselves, and if the uploader does not give us contact information, we cannot even let them know their images broke.
